README.md

```diff
@@ -1 +1,46 @@
-# hello-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] It's better if you replace this text with the project's name (Hello Microverse) instead of putting the project's name below it in small text. It would look better as the main header of the README file 👍